mapper/thin_edge_json/src/serialize.rs

```diff
@@ -71,6 +74,11 @@ impl ThinEdgeJsonSerializer {
         self.end()?;
         Ok(self.buffer)
     }
+
+    pub fn into_string(self) -> Result<String, ThinEdgeJsonSerializationError> {
```

Comment:
With PR #281 this will become simpler. No need to deal with invalid UTF8.

Reply:
Sure